### PR TITLE
Updates for Scala 2.13

### PR DESCRIPTION
**Pull request by travisbrown**
The bincompat issue is the same as https://github.com/sangria-graphql-org/sangria-ion/pull/1—there's a more specific type inferred for an implementation of a `InputMarshaller` method that causes issues on 2.13.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -58,6 +57,6 @@ startYear := Some(2016)
 organizationHomepage := Some(url("https://github.com/sangria-graphql"))
 developers := Developer("OlegIlyenko", "Oleg Ilyenko", "", url("https://github.com/OlegIlyenko")) :: Nil
 scmInfo := Some(ScmInfo(
-  browseUrl = url("https://github.com/sangria-graphql/sangria-msgpack.git"),
-  connection = "scm:git:git@github.com:sangria-graphql/sangria-msgpack.git"
-))
\ No newline at end of file
+  browseUrl = url("https://github.com/sangria-graphql-org/sangria-msgpack.git"),
+  connection = "scm:git:git@github.com:sangria-graphql-org/sangria-msgpack.git"
+))
```
Comment (yanns) [inline build.sbt:63]: I guess we'll also need:
```
releaseCrossBuild := true
releasePublishArtifactsAction := PgpKeys.publishSigned.value
```

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+[![Build Status](https://travis-ci.org/sangria-graphql-org/sangria-msgpack.svg?branch=master)](https://travis-ci.org/sangria-graphql-org/sangria-msgpack) [![Coverage Status](http://coveralls.io/repos/sangria-graphql-org/sangria-msgpack/badge.svg?branch=master&service=github)](http://coveralls.io/github/sangria-graphql-org/sangria-msgpack?branch=master) [![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.sangria-graphql/sangria-msgpack_2.11/badge.svg)](https://maven-badges.herokuapp.com/maven-central/org.sangria-graphql/sangria-msgpack_2.11) [![License](http://img.shields.io/:license-Apache%202-brightgreen.svg)](http://www.apache.org/licenses/LICENSE-2.0.txt) [![Join the chat at https://gitter.im/sangria-graphql/sangria](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/sangria-graphql/sangria?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```
Comment (yanns) [inline README.md]: for info, we can break them on different lines as markdown will still display them next to each other.
It makes clearer diff IMO.
(just for info)

File: build.sbt (hunk 1 above)
Comment (travisbrown) [reply build.sbt:63]: Thanks, good catch—trying to do these between other things and missed that.

File: README.md (hunk 2 above)
Comment (travisbrown) [reply README.md]: Agreed, and done.

**Review by yanns — APPROVED** [review]
🏆

**Pull request merged**